.travis.yml

```diff
@@ -51,4 +51,4 @@ matrix:
   allow_failures:
     - jdk: openjdk11
 script:
-  - mvnp clean install -B -DskipChecks=true -DskipTests=true $(extra_maven_opts)
+  - MAVEN_OPTS="-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=WARN -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN" mvnp clean install -B -DskipChecks=true -DskipTests=true $(extra_maven_opts)
```

Comment:
'-Dorg.slf4j.simpleLogger.defaultLogLevel=WARN' is not making any difference in output in my local build. The other one works as expected.

Reply:
I tested it locally. But with "warn" instead of WARN